pkg/rulefmt/rulefmt.go

```diff
@@ -26,13 +26,14 @@ import (
 
 // Error represents semantical errors on parsing rule groups.
 type Error struct {
-	Group string
-	Rule  int
-	Err   error
+	Group    string
+	Rule     int
+	RuleName string
+	Err      error
 }
 
 func (err *Error) Error() string {
-	return errors.Wrapf(err.Err, "group %q, rule %d", err.Group, err.Rule).Error()
+	return errors.Wrapf(err.Err, "group %q, rule %d, %s", err.Group, err.Rule, err.RuleName).Error()
```

Comment:
I'd use %q so that it ends up in quotes. Some alert/rule names may be confused as part of the error text otherwise.